relay-server/src/actors/store.rs

```diff
@@ -1136,24 +1133,53 @@ struct CheckInKafkaMessage {
     retention_days: u16,
 }
 
-#[derive(Debug, Serialize)]
+#[derive(Debug, Deserialize, Serialize)]
 struct SpanKafkaMessage<'a> {
-    /// Time at which the event was received by Relay. Not to be confused with `start_timestamp_ms`.
-    start_time: u64,
+    #[serde(skip_serializing)]
+    start_timestamp: f64,
+    #[serde(rename(deserialize = "timestamp"), skip_serializing)]
+    end_timestamp: f64,
+
+    description: &'a str,
+    #[serde(default)]
+    duration_ms: u32,
     /// The ID of the transaction event associated to this span, if any.
-    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(default, skip_serializing_if = "Option::is_none")]
     event_id: Option<EventId>,
-    /// The numeric ID of the organization.
-    organization_id: u64,
+    #[serde(rename(deserialize = "exclusive_time"))]
+    exclusive_time_ms: f64,
+    is_segment: bool,
+
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    measurements: Option<&'a RawValue>,
+    #[serde(
+        default,
+        rename = "_metrics_summary",
+        skip_serializing_if = "Option::is_none"
+    )]
+    metrics_summary: Option<&'a RawValue>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    parent_span_id: Option<&'a str>,
     /// The numeric ID of the project.
-    project_id: ProjectId,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    profile_id: Option<&'a str>,
+    #[serde(default)]
+    project_id: u64,
+    /// Time at which the event was received by Relay. Not to be confused with `start_timestamp_ms`.
+    received: f64,
     /// Number of days until these data should be deleted.
+    #[serde(default)]
     retention_days: u16,
-    /// Fields from the original span payload.
-    /// See [`relay-event-schema::protocol::span::Span`] for schema.
-    ///
-    /// By using a [`RawValue`] here, we can embed the span's JSON without additional parsing.
-    span: &'a RawValue,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    segment_id: Option<&'a str>,
+    #[serde(default, skip_serializing_if = "BTreeMap::is_empty")]
+    sentry_tags: BTreeMap<&'a str, &'a str>,
```

Comment:
Any reason this is not a `RawValue`?  Is it because of the `skip_serializing_if`? Could we live without `skip_serializing_if`?